src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java

```diff
@@ -53,6 +53,36 @@ protected void registerDefaultRecipes(List<ItemStack> recipes) {
         recipes.add(SlimefunItems.GOLD_4K);
         recipes.add(SlimefunItems.GOLD_DUST);
 
+        recipes.add(SlimefunItems.GOLD_6K);
+        recipes.add(SlimefunItems.GOLD_DUST);
+
+        recipes.add(SlimefunItems.GOLD_8K);
+        recipes.add(SlimefunItems.GOLD_DUST);
+
+        recipes.add(SlimefunItems.GOLD_10K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 2));
+
+        recipes.add(SlimefunItems.GOLD_12K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 2));
+
+        recipes.add(SlimefunItems.GOLD_14K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 2));
+
+        recipes.add(SlimefunItems.GOLD_16K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 2));
+
+        recipes.add(SlimefunItems.GOLD_18K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 3));
+
+        recipes.add(SlimefunItems.GOLD_20K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 3));
+
+        recipes.add(SlimefunItems.GOLD_22K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 3));
+
+        recipes.add(SlimefunItems.GOLD_24K);
+        recipes.add(new SlimefunItemStack(SlimefunItems.GOLD_DUST, 3));
```

Comment:
Not sure about the balancing aspects of this, whether it should be more or if it's balanced as is.
I can also see lots of people potentially reporting this inconsistency as a bug :/

Reply:
It's not exactly inconsistent. It's actually carat / 8.

Reply:
The suggestion on the Discord suggested 1 dust, no matter how much carat.
I thought this doesn't make all too much sense, so as Linox mentioned, I did carat / 8 and rounded up

Reply:
However, when I was implementing this it did look like whoever had already implemented the idea of a 4carat gold returning 1 dust, had the idea in mind to equally return what was needed to make it

Reply:
You could make it carat / 4 so every 2 dust returns 1. This would be enough to stay away from the bug reports. You could also add only 1 of the gold ingots to sf guide recipes and say that this returns half the carat the gold has.

Reply:
Also I remember there was also 4 carat. Maybe add that too?

Reply:
4 carat to one gold dust has always been a thing, the recipe already exists

Reply:
And how exactly would I go about this:
> You could also add only 1 of the gold ingots to sf guide recipes and say that this returns half the carat the gold has.
?

Reply:
> 4 carat to one gold dust has always been a thing, the recipe already exists

Oh... Lol

Reply:
> And how exactly would I go about this:
> > You could also add only 1 of the gold ingots to sf guide recipes and say that this returns half the carat the gold has.
> 
> edit: O you mean like a small for loop that gets the carat and returns half of the carat the ingot is worth?

No I mean take this as a suggestion:

- Instead of dividing by 8, divide by 4.
- Add a display recipe for ore grinder to sf guide stating that equation.

Reply:
> 
> 
> > And how exactly would I go about this:
> > > You could also add only 1 of the gold ingots to sf guide recipes and say that this returns half the carat the gold has.
> > 
> > 
> > edit: O you mean like a small for loop that gets the carat and returns half of the carat the ingot is worth?
> 
> No I mean take this as a suggestion:
> 
>     * Instead of dividing by 8, divide by 4.
> 
>     * Add a display recipe for ore grinder to sf guide stating that equation.

Sure, Ill do that
But uhm, by adding those recipes they automatically make displayrecipes for every ingot so wouldnt it be kind of useless?
Or is there a way to prevent the recipes.add() thing from making them displayrecipes?

Reply:
@LinoxGH ^

Reply:
> @LinoxGH ^ 

Idk

Reply:
Well I dont think its easily done, so Ill just say this is resolved

Reply:
I'm just gonna skip this long conversation and look at the file diff if that's okay for you? XD